.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -170,39 +170,61 @@ AddType text/vtt                            vtt
+    RewriteRule rosa/rosa_getting_started/rosa_getting_started_iam/rosa-accessing-cluster.html rosa/rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa-config-identity-providers.html rosa/rosa_install_access_delete_clusters/rosa-sts-config-identity-providers.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa_getting_started_iam/rosa-config-identity-providers.html rosa/rosa_install_access_delete_clusters/rosa-sts-config-identity-providers.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa-deleting-access-cluster.html rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-deleting-access-cluster.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa-deleting-cluster.html rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-deleting-cluster.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa-quickstart.html rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-quickstart.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa_getting_started_iam/?(.*)$ rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/$1 [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa-sts-about-iam-resources.html rosa/rosa_architecture/rosa-sts-about-iam-resources.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa-sts-required-aws-service-quotas.html rosa/rosa_planning/rosa-sts-required-aws-service-quotas.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/rosa-sts-setting-up-environment.html rosa/rosa_planning/rosa-sts-setting-up-environment.html [NE,R=302]
+    RewriteRule rosa/rosa_getting_started/(rosa-getting-started|rosa-sts-getting-started-workflow).html - [L]
+    RewriteRule rosa/rosa_getting_started/?(.*)$ rosa/rosa_install_access_delete_clusters/$1 [NE,R=302]
+
+    RewriteRule rosa/logging/?(.*)$ rosa/rosa_cluster_admin/rosa_logging/$1 [NE,R=302]
+
+    RewriteRule rosa/monitoring/osd?(.*)$ rosa/rosa_cluster_admin/rosa_monitoring/rosa$1 [NE,R=302]
+
+    RewriteRule rosa/cloud_infrastructure_access/?(.*)$ rosa/rosa_cluster_admin/cloud_infrastructure_access/$1 [NE,R=302]
+
+    RewriteRule rosa/nodes/nodes-machinepools-about.html rosa/rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.html [NE,R=302]
+    RewriteRule rosa/nodes/rosa-managing-worker-nodes.html rosa/rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html [NE,R=302]
+    RewriteRule rosa/nodes/nodes-about-autoscaling-nodes.html rosa/rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.html [NE,R=302]
+
+    # ROSA UI short-term redirect https://issues.redhat.com/browse/OSDOCS-3511
+    RewriteRule rosa/post_installation_configuration/machine-configuration-tasks.html rosa/rosa_getting_started/rosa-getting-started.html#rosa-getting-started-configure-an-idp-and-grant-access_rosa-getting-started [NE,R=302]
```

Comment:
@bmcelvee and @bscott-rh I have added this additional redirect entry as a short-term solution for https://issues.redhat.com/browse/OSDOCS-3511. It directs to a section within an assembly, which I have seen used elsewhere in the redirect file. Please let me know if the redirect looks good. Thanks!

Reply:
LGTM.

Reply:
Thank you!